test/ngAnimate/animateCssSpec.js

```diff
@@ -316,6 +316,96 @@ describe("ngAnimate $animateCss", function() {
           options = { event: 'enter', structural: true };
         }));
 
+        it("should always return an object even if no animation is detected",
+          inject(function($animateCss) {
+
+          ss.addRule('.some-animation', 'background:red;');
+
+          element.addClass('some-animation');
+          var animator = $animateCss(element, options);
+
+          expect(animator).toBeTruthy();
+          expect(animator.start).toBeTruthy();
```

Comment:
I'd suggest to use the test to enforce the API contract as well:
```js
expect(animator.start instanceof Function).toBe(true);
// or
expect(angular.isFunction(animator.start)).toBe(true);
```

The same would apply for the other methods being testes as `toBeTruthy`